examples/jsm/nodes/materials/MeshStandardNodeMaterial.d.ts

```diff
@@ -1,7 +1,23 @@
+import { Color } from '../../../../src/math/Color';
+import { Texture } from '../../../../src/textures/Texture';
+import { CubeTexture } from '../../../../src/textures/CubeTexture';
+import { Vector2 } from '../../../../src/math/Vector2';
+
+import { Node } from '../core/Node';
 import { NodeMaterial } from './NodeMaterial';
 
 export class MeshStandardNodeMaterial extends NodeMaterial {
 
 	constructor();
 
+	color: Color | Node;
```

Comment:
I guess this should be `Color | Node | string | number;`.

Reply:
Ok, but `number`? Why is a number a valid color?

Reply:
For example `0xff0000`.

Reply:
> You can use color definitions like 0xff0000.

I think that this feature is available only in r8 ( https://github.com/mrdoob/three.js/pull/17265 )

Reply:
Oh, sorry. I've mixed this with the default materials^^.